packages/cli/configuration/fern/definition/docs.yml

```diff
@@ -20,13 +20,26 @@ types:
       - jvm
       - ts
       - js
+
+  AnalyticsConfig:
+    properties:
+      segment: optional<SegmentConfig>
+
+  SegmentConfig:
+    properties:
+      writeKey: string
       
   DocsConfiguration:
     properties:
       instances: list<DocsInstance>
       title:
         type: optional<string>
         docs: used as tab bar title, and in the navbar if no logo is defined
+      analyticsConfig: 
```

Comment:
nit: can we call this `analytics` so that users in the `docs.yml` just have to write out `analytics` 

```
analytics:
  segment: 
     write-key: "..."
```